rio_cogeo/scripts/cli.py

```diff
@@ -279,6 +269,21 @@ def create(
     elif web_optimized:
         tilematrixset = morecantile.tms.get("WebMercatorQuad")
 
+    if tilematrixset:
+        blocksize = blocksize or min(
+            tilematrixset.matrix(tilematrixset.minzoom).tileHeight,
+            tilematrixset.matrix(tilematrixset.minzoom).tileWidth,
+        )
+        overview_blocksize = overview_blocksize or blocksize
```

Comment:
1. use TMS tileHeight or tileWidth if `blocksize` is not provided 
2. overview blocksize default to `blocksize` if not provided (only when there is a tilematrixset option)

Reply:
`tilematrix.matrix(tilematrixset.minzoom)` gives the lowest-resolution matrix, whereas `tilematrix.matrix(tilematrixset.minzoom)` returns the best level of details. Therefore, I would tend to define blocksize using `tilematrix.matrix(tilematrixset.maxzoom)`, and keep the `minzoom` one for overview block size.

Now, I am not sure it matters in practice, because I have not seen any tile-matrix set changing tile size between levels (and in such case, looking at only the min and max level would not be enough anyway).

Reply:
most TMS should have the same tileHeight and tileWidth at all level (at least for the common grids), ideally we would select the blocksize for the zoom level the data cover but we can't get this in the CLI